backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxUtils.java

```diff
@@ -195,15 +230,22 @@ private static void setText(XWPFRun run, String text) {
         }
     }
 
-    private static void addFormattedText(XWPFRun run, String text, String fontFamily, int fontSize, boolean bold) {
+    private static void addFormattedText(XWPFRun run, String text, String fontFamily, int fontSize, boolean bold, String rrggbbColor) {
         run.setFontSize(fontSize);
         run.setFontFamily(fontFamily);
         run.setBold(bold);
+        if (rrggbbColor != null) {
+            run.setColor(rrggbbColor);
+        }
         setText(run, text);
     }
 
+    private static void addFormattedText(XWPFRun run, String text, int fontSize, boolean bold, String rrggbbColor) {
+        addFormattedText(run, text, "Calibri", fontSize, bold, rrggbbColor);
```

Comment:
magic string `"Calibri"`. Is the font/fontfamily Callibri present on every OS?

Reply:
RIght, I extracted a constant now. Don't ask me about font family availability. I can only point to "prior art", i.e. this existed and seemed to work before my change.